polaris/polaris/sep24/deposit.py

```diff
@@ -149,6 +149,13 @@ def post_interactive_deposit(request: Request) -> Response:
             )
             or {}
         )
+        if registered_scripts_func != calculate_fee:
```

Comment:
@JakeUrban This seems incorrect.
Maybe it should be `if registered_scripts_fund != scripts`?

Reply:
`scripts` is the keyword argument to the `register_integrations()`. `calculate_fee` is actually the default function Polaris calls if no function was passed to `register_integrations()` `scripts`.

Reply:
Good catch, but it should actually be
```
registered_fee_func != calculate_fee
```

Reply:
Actually nevermind, you're right completely!